<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormJobManagement</class>
 <widget class="QDialog" name="FormJobManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>308</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>804</width>
    <height>597</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestion des travaux</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du travail :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Liste des numéros de billets :&lt;/p&gt;&lt;p&gt;(séparé par des virgules)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditTicketIds">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 173, 44, 255), stop:1 rgba(44, 44, 44, 255));</string>
   </property>
   <property name="text">
    <string>Gestion des travaux</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>249</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="text">
       <string>&amp;OK</string>
      </property>
      <property name="icon">
       <iconset resource="formjobmanagement.qrc">
        <normaloff>:/FormJobManagement/Ok22.png</normaloff>:/FormJobManagement/Ok22.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>&amp;Annuler</string>
      </property>
      <property name="icon">
       <iconset resource="formjobmanagement.qrc">
        <normaloff>:/FormJobManagement/Cancel22.png</normaloff>:/FormJobManagement/Cancel22.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="globalformressources.qrc"/>
  <include location="formjobmanagement.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>FormJobManagement</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
